Iterate.py

```diff
@@ -49,18 +49,16 @@ def LengFunc(y, s, si, st):
         if s >S[Xpoint]:
             # The second term here converts the x point qpar to a radial heat source acting between midplane and the xpoint
             try:
-                dqoverBds = ((nu**2*Tu**2)/T**2)*cz*Lfunc(T) - qradial * fieldValue / B(S[Xpoint]) # account for flux expansion to Xpoint
+                dqoverBds = ((nu**2*Tu**2)/T**2)*cz*Lfunc(T)/fieldValue - qradial/fieldValue #/fieldValue * fieldValue / B(S[Xpoint]) # account for flux expansion to Xpoint

```

Comment:
This puts the division by B inline to avoid confusion and fixes qradial which previously led to up to a 50% error on qpar at the X-point.